src/panels/profile/ha-change-password-card.ts

```diff
@@ -163,6 +170,44 @@ class HaChangePasswordCard extends LitElement {
     this._statusMsg = this.hass.localize(
       "ui.panel.profile.change_password.success"
     );
+
+    if (
+      this.refreshTokens &&
+      this.refreshTokens.length > 1 &&
+      (await showConfirmationDialog(this, {
+        title: this.hass.localize(
+          "ui.panel.profile.change_password.logout_all_sessions"
+        ),
+        text: this.hass.localize(
+          "ui.panel.profile.change_password.logout_all_sessions_text"
+        ),
+        dismissText: this.hass.localize("ui.common.no"),
+        confirmText: this.hass.localize("ui.common.yes"),
+        destructive: true,
+      }))
+    ) {
+      const promises = this.refreshTokens
+        .filter((token) => token.type === "normal" && !token.is_current)
+        .map((token) =>
+          this.hass.callWS({
+            type: "auth/delete_refresh_token",
+            refresh_token_id: token.id,
+          })
+        );
```

Comment:
Should we let the backend handle this instead?

Reply:
🤷‍♂️  doesn't really matter I think, but we could